07-routing-gossip.md

```diff
@@ -226,10 +226,9 @@ nodes not associated with an already known channel are ignored.
    * [`2`:`addrlen`]
    * [`addrlen`:`addresses`]
 
-The `timestamp` allows ordering of messages, in the case of multiple
-announcements; the `rgb_color` and `alias` allow intelligence services to give
-their nodes cool monikers like 'IRATEMONK' and 'WISTFULTOLL' and to use the
-color black.
+`timestamp` allows for the ordering of messages, in the case of multiple
+announcements. `rgb_color` and `alias` allow intelligence services to assign
+nodes colors (e.g. black) and cool monikers like 'IRATEMONK' and 'WISTFULTOLL'.
```

Comment:
It's a niggling bit, but I always prefer parallel lists (e.g., "assign nodes colors like black and ..."

Reply:
your wish is my command